CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## UNRELEASED
 
+### Adds
+
+* Adds support for `permission` property in `addContextOperation`, this one can be a string or an array. Validates `addContextOperation` configuration.
```

Comment:
This is great, but a lot of these conditions are not really Apostrophe permission names. So right now it looks too much like something it is not.

I think we should just call this property `conditions`, require it to be an array, and make the values in the array the actual property names: `canPublish`, `canDismissSubmission`, etc.

This will avoid misleading the developer into thinking they can test any Apostrophe permission, and eliminate the need for string manipulation.

Reply:
Ok yes makes sense to me 👍🏼